File: kumoh-code-2017/Java/#1 기본/#2 수업/소스코드/7. Account 클래스/실습5(완성본)/Account.java
public class Account
{
	private String account_number; // 계좌번호
	private String Owner; // 예금주
	private long balance; // 잔고
	
	public Account(String account_number, String Owner, long balance)
	{
		this.account_number = account_number;
		this.Owner = Owner;
		this.balance = balance;
	}
	
	public String getAccountNumber(){return account_number;}
	public String getOwner(){return Owner;}
	public long getBalance(){return balance;}
	public void setBalance(long balance){this.balance = balance;}
}

File: kumoh-code-2017/C++/#2 설계/과제15 은행관리 프로그램/Account.cpp
#include"Account.h"

Account::Account(){
	accountNumber = 0;
	password = 0;
	balance = 0;
}

Account::Account(int accountNumber, int password, int balance){
	this->accountNumber = accountNumber;
	this->password = password;
	this->balance = balance;
}

bool Account::checkPassword(int password){
	if (this->password == password){
		return true;
	}
	else
		return false;
}

int Account::setBalance(int balance){ //입출금시 입력받은값을 분석하여 stop인지 아닌지를 구별
	Screen screen;
	this->balance = balance;
	if (balance != -999){
		return balance;
	}
}

void Account::credit(int money){
	balance = balance+money;
}

void Account::debit(int money){
	balance = balance-money;
}

